### PR TITLE
Chapter4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by VietHungLe**
Ready

### CODE REVIEW

**Comment by VietHungLe** [comment]
Ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (thanhhungchu95) [inline app/helpers/application_helper.rb]: i18n

**Comment by VietHungLe** [comment]
Ready

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <title><%= yield :title %> | <%= t "ror_sampleapp" %></title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+
```
Comment (thanhhungchu95) [inline app/views/layouts/application.html.erb]: xóa space thừa

**Review by thanhhungchu95 — COMMENTED** [review]
fix xong merged nha men

**Pull request merged**